src/express-middleware/image-proxy/file-cache.js

```diff
@@ -14,18 +15,19 @@ function isNumber(val) {
 
 class FileCache {
   constructor(opts = defaultOpts) {
-    this.basePath = opts.basePath;
+    this.basePath = process.env.VOLTO_FILE_CACHE_BASE_PATH || opts.basePath;
```

Comment:
@silviubogan  we can remove `process.env.VOLTO_FILE_CACHE_BASE_PATH` from here? below we're resolving path already, I think.

Reply:
@nileshgulia1 I think no because below we are resolving the path relative to the current working directory which is the root directory of Volto, and `VOLTO_FILE_CACHE_BASE_PATH` preferably contains a path relative to that root directory although it can contain absolute paths too.